machines/4x4/Makefile.machine.include

```diff
@@ -14,9 +14,8 @@ BSG_MACHINE_DRAM_INCLUDED             = 1
 BSG_MACHINE_MAX_EPA_WIDTH             = 28
 
 # supported memory cfg
-# e_infinite_mem
```

Comment:
I feel like makefiles in this directory shouldn't switch between memory configurations like this. maybe have a subdirectory for each config?